src/client/sandbox/node/window.js

```diff
@@ -95,6 +110,18 @@ export default class WindowSandbox extends SandboxBase {
             return false;
         };
 
+        window.setTimeout = function () {
```

Comment:
Let's move it to the `timers` sandbox. It's not cool to have overrides in two places. And you will not need to re-override timers in `timers` sandbox for IE then.